tests/README.md

```diff
@@ -71,11 +69,19 @@ cd tests/servers
 git submodule update --remote --recursive
 ```
 
+First, setup the docker image required for running server-requiring tests:
+```
+docker run --rm -d -p 5432:5432 --name postgres-coordinatord -e POSTGRES_PASSWORD=test -e POSTGRES_USER=test -e POSTGRES_DB=coordinator_db postgres:alpine
+```
+
 To run the server-requiring tests, pass the postgres credentials to the framework:
 ```
 POSTGRES_USER="test" POSTGRES_PASS="test" TEST_DEBUG=1 pytest -vvv -n8 --timeout=1800
 ```
 
+It is plausible that your functional tests encounter timeout error, you can 
+increase it by setting the environment variable of your shell by 
+`export TIMEOUT=120` command and run on single thread.
```

Comment:
I don't think we should recommend `export`ing variables
```suggestion
It is plausible that your functional tests encounter timeout error, you can 
increase it by setting the `TIMEOUT` environment variable to a higher value
(for instance `TIMEOUT=120`). Running in single-threaded mode (by not
specifying the `-n` parameter) might also help.
```

Reply:
Also, compiling in release mode and setting `REVAULTD_PATH=$PWD/target/release/revaultd` is a pretty good way to avoid timeouts.